README.md

```diff
@@ -15,22 +17,83 @@ Create a challenge. Ready? Brainstorm. mindwendel helps you to easily brainstorm
 ![](docs/screenshot2.png)
 
 ## Use-cases
+
 Brainstorm ...
+
 - ... new business ideas
 - ... solutions for a problem
 - ... what to eat tonight
-- ... 
+- ...
 
 ## Getting Started
 
-### Prerequisites
+mindwendel can be run just about anywhere
+
+ <!-- TODO: Add an installation guide with detailed instructions for various deployments. -->
```

Comment:
what kind of guide do you have in mind? I think our docker setup is pretty straight forward.